File: AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/nativeads/NativeAdDemoMenuActivity.java
package com.applovin.enterprise.apps.demoapp.ads.applovin.nativeads;

import android.content.Intent;

import com.applovin.enterprise.apps.demoapp.ads.applovin.DemoMenuActivity;
import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem;

public class NativeAdDemoMenuActivity
        extends DemoMenuActivity
{
    @Override
    protected DemoMenuItem[] getListViewContents()
    {
        return new DemoMenuItem[] {
                new DemoMenuItem( "Single Ad", new Intent( this, NativeAdCarouselUIActivity.class ) ),
                new DemoMenuItem( "Multiple Ads", new Intent( this, NativeAdRecyclerViewActivity.class ) )
        };
    }
}

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/nativeads/NativeAdDemoMenuActivity.kt
package com.applovin.enterprise.apps.demoapp.ads.applovin.nativeads

import android.content.Intent
import com.applovin.enterprise.apps.demoapp.ads.DemoMenuActivity
import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem

class NativeAdDemoMenuActivity : DemoMenuActivity()
{
    override fun getListViewContents(): Array<DemoMenuItem> = arrayOf(
            DemoMenuItem("Single Ad", Intent(this, NativeAdCarouselUIActivity::class.java)),
            DemoMenuItem("Multiple Ads", Intent(this, NativeAdRecyclerViewActivity::class.java))
    )
}
